src/datasets/features/audio.py

```diff
@@ -91,10 +91,11 @@ def decode_example(self, value: dict) -> dict:
             raise RuntimeError("Decoding is disabled for this feature. Please use Audio(decode=True) instead.")
 
         path, file = (value["path"], BytesIO(value["bytes"])) if value["bytes"] is not None else (value["path"], None)
+        extension = path.split(".")[-1]
```

Comment:
The path can be None here:
```suggestion
        extension = path.split(".")[-1] if path is not None else None
```